### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by kulveersingh** [Enterprise-Content-Management/documentum-rest-sample-android#1]
04-06 17:05:30.080 2465-2465/com.opentext.documentum.rest.sample.android D/LoginObservables: java.lang.IllegalStateException --> Could not create URI object: Illegal character in scheme name at index 4: http%20://test/dctm-rest/repositories/services

URL entered => http://myserver.8080/dctm-rest
User id 
Password
I tested the REST services using RESTLET chrome plugin, they are workign fine. I get this error in the emulator. Can you help me what could the issue ?

**Comment by gentlewind** [comment]
Hi @kulveersingh , the error indicates that there was a blank character (encoded as `%20`) between `http` and `://` in your input.
> http%20://test/dctm-rest/repositories/services

Would you please double check the input URL? Or capture your screenshot for a look?

**Comment by gentlewind** [comment]
A right service URL input should be like: `http://localhost:8080/dctm-rest`

**Comment by kulveersingh** [comment]
I got rid of that error but I get another one . Can you tell me what exact version you tested it again for REST services ?

Error now is 
04-07 10:57:39.960 17261-17440/com.opentext.documentum.rest.sample.android I/art: Unable to resolve java.lang.Class<com.emc.documentum.rest.client.sample.model.LinkableBase> annotation class 414
04-07 10:57:40.041 17261-17440/com.opentext.documentum.rest.sample.android I/System.out:   [debug] Response status: 200
04-07 10:57:40.041 17261-17440/com.opentext.documentum.rest.sample.android I/System.out:   [debug] Response headers: {X-Powered-By=[Servlet/3.0], Cache-Control=[max-age=31536000], Content-Type=[application/home+json;charset=UTF-8], Content-Language=[en-US], Transfer-Encoding=[chunked], Date=[Fri, 07 Apr 2017 14:57:39 GMT]}
04-07 10:57:40.041 17261-17440/com.opentext.documentum.rest.sample.android I/System.out:   [debug] Resource URI: http://11.11.11.123:9081/dctm-rest/repositories
04-07 10:57:40.041 17261-17440/com.opentext.documentum.rest.sample.android I/System.out:   [debug] HTTP method: GET
04-07 10:57:40.041 17261-17440/com.opentext.documentum.rest.sample.android I/System.out:   [debug] Request headers: {Accept=[application/vnd.emc.documentum+json]}
04-07 10:57:40.041 17261-17440/com.opentext.documentum.rest.sample.android I/System.out:   [debug] Expected response body class: RepositoryFeed
04-07 10:57:40.041 17261-17440/com.opentext.documentum.rest.sample.android I/System.out:   [debug] Request parameters: 
04-07 10:57:40.042 17261-17440/com.opentext.documentum.rest.sample.android I/System.out:   [debug] Authenticate with Basic ssssssAa18yMDE1RGU=
04-07 10:57:40.042 17261-17440/com.opentext.documentum.rest.sample.android I/System.out:   [debug] Sending GET request to http://11.11.11.123:9081/dctm-rest/repositories
04-07 10:57:40.043 17261-17440/com.opentext.documentum.rest.sample.android I/art: Unable to resolve java.lang.Class<com.emc.documentum.rest.client.sample.model.LinkableBase> annotation class 414
04-07 10:57:40.049 17261-17440/com.opentext.documentum.rest.sample.android I/art: Unable to resolve java.lang.Class<com.emc.documentum.rest.client.sample.model.LinkableBase> annotation class 414
04-07 10:57:40.064 17261-17440/com.opentext.documentum.rest.sample.android I/art: Unable to resolve java.lang.Class<com.emc.documentum.rest.client.sample.model.LinkableBase> annotation class 414
04-07 10:57:40.140 17261-17261/com.opentext.documentum.rest.sample.android D/LoginObservables: org.springframework.http.converter.HttpMessageNotReadableException --> Could not read JSON: Unrecognized field "content-type" (class com.emc.documentum.rest.client.sample.model.json.JsonRepository), not marked as ignorable (7 known properties: "servers", "src", "links", "type", "id", "description", "name"])
                                                                                                   at [Source: org.apache.http.conn.EofSensorInputStreamHC4@58d5556; line: 1, column: 426] (through reference chain: com.emc.documentum.rest.client.sample.model.json.RepositoryFeed["entries"]->java.util.ArrayList[0]->com.emc.documentum.rest.client.sample.model.json.JsonEntry["content"]->com.emc.documentum.rest.client.sample.model.json.JsonRepository["content-type"]); nested exception is com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "content-type" (class com.emc.documentum.rest.client.sample.model.json.JsonRepository), not marked as ignorable (7 known properties: "servers", "src", "links", "type", "id", "description", "name"])
                                                                                                   at [Source: org.apache.http.conn.EofSensorInputStreamHC4@58d5556; line: 1, column: 426] (through reference chain: com.emc.documentum.rest.client.sample.model.json.RepositoryFeed["entries"]->java.util.ArrayList[0]->com.emc.documentum.rest.client.sample.model.json.JsonEntry["content"]->com.emc.documentum.rest.client.sample.model.json.JsonRepository["content-type"])

**Comment by gentlewind** [comment]
REST 7.2 and 7.3 were tested. It could be possible that this error occurred on earlier versions because of this change on atom entry (documented in REST ref guide).
```
Note: In release 7.1, the property indicating the MIME type was named as content-type. This
name is deprecated and replaced with type.

Example 2-164. Representation in 7.1
content: {
content-type: "application/json"
src: "http://localhost:8080/dctm-rest/repositories/REPO/types/dm_acs_config"
}

Example 2-165. Current Representation
content: {
type: "application/json"
src: "http://localhost:8080/dctm-rest/repositories/REPO/types/dm_acs_config"
}
```
* If you were using REST 7.0 or 7.1, please upgrade to REST 7.2 which is backward compatible to Content Server earliest to 6.7SP2, or REST 7.3 which is backward compatible to Content Server 7.1.
* We will also check the Java client project to see whether this can be addressed by the client library (next week).

**Comment by gentlewind** [comment]
Hi @kulveersingh , the issue is resolved by updating the java client, in commit 8a3bf67e8bf44e5d623766217b75f8929bd9f44c. Please checkout the latest android code.

I'll close this issue in 2 days if there is no further update.
